week2_classwork_24Jan .ipynb

```diff
@@ -0,0 +1,2159 @@
+{
```

Comment:
<p>where is the result for this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/0Swathi/txt_based_analysis/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>